wmn-data.json

```diff
@@ -6337,7 +6284,7 @@
         "strip_bad_char" : ".",

```

Comment:
Looks like the `uri_check` is not great anymore since this site has archived people's blogs. Let's change it to use `https://help.xanga.com/profile/{account}` for the `uri_check`. Example users are `brooklynheart` and `eugenia`. 

Please adjust other site entries to reflect the new URL responses